osf/management/commands/test_publish.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
```

Comment:
is `test_publish` meant to be kept and used, or was it just for testing while coding?

Reply:
I thought it might be a nice tools for Devops.

Reply:
Maybe a rename to something without `test` in the name, since it's not limited to just publishing fake test data?